File: Structure-syntax/10 Array Strings/Java/Main.java
/*
* Java program
*/
public class Main {
  public static void main(String [] argv) {

     int [] A1 = new int [2];
     int [] A2 = new int [2];
     A1 [0] = 0;
     A1 [1] = 1;
     System.arraycopy(A1, 0, A2, 0, A1.length);

     System.out.println("");
     System.out.print(A2[0]);
     System.out.print(" ");
     System.out.println(A2[1]);
     System.out.println("");
  }
}

File: Structure-syntax/10 Array Strings/C++/src/main.cpp
/*
 * C++ program
 */
#include <iostream>
using namespace std;
int main(int argc, const char* argv[])  {
   int A1 [2];
   int A2 [2];
   A1 [0] = 0;
   A1[1] = 1;
   cout << endl;
   for (int i = 0; i < 2; ++i) {
       A2[i] = A1[i];
       cout << A2[i] << " ";
   }
   cout << endl;
   cout << endl;
}

